docs/layouts/partials/site-footer.html

```diff
@@ -0,0 +1,10 @@
+<footer class="bg-primary-color-dark ph4-ns pt4 relative w-100" role="contentinfo">
+      <div class="b f3 center w4 light-gray mb3 nested-links tc">
+          <a href="https://github.com/ovh/cds/graphs/contributors" class="link"><img src="images/cds-transparent.png"> Contributors</a><br />
+      </div>
+    <div class="f7 gray mb5 mb0-ns ph3 w-100"> 
+      <p class="dib mr4">The CDS logos are copyrighted.</p>
+      <p class="dib">Icons made by <a href="https://www.flaticon.com/authors/eucalyp"  class="dim link light-gray pv2" title="Eucalyp">Eucalyp</a> from <a class="dim link light-gray pv2" href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a> licensed by <a class="dim link light-gray pv2" href="http://creativecommons.org/licenses/by/3.0/" title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a></p>
```

Comment:
it seems it should be a link with the text "`designed by {Author's Name} from Flaticon"`
 regarding https://file000.flaticon.com/downloads/license/license.pdf